tooling/minimize/src/main.rs

```diff
@@ -52,12 +63,15 @@ mod rvalue;
 mod constant;
 
 mod get;
+
 use get::get_mini;
 
 mod chunks;
+
 use chunks::calc_chunks;
 
 mod enums;
+
```

Comment:
Why did you reformat this?

Reply:
I just ran `cargo fmt` or it's possible that my IDE messed it up. let me fix

Reply:
Yeah, it was my IDE. I've been playing around with RustRover, and it has some rough edges. I reverted it.